doc/devices/numpy_wavefunction.rst

```diff
@@ -0,0 +1,77 @@
+The Numpy-Wavefunction device
+=============================
+
+The ``forest.numpy_wavefunction`` device provides an interface between PennyLane
+and the pyQuil `NumPy wavefunction simulator <http://docs.rigetti.com/en/stable/wavefunction_simulator.html>`_.
+
+Because the NumPy wavefunction simulator allows access and manipulation of the underlying
+quantum state vector, ``forest.numpy_wavefunction`` is able to support the full
+suite of PennyLane and Quil quantum operations and observables.
+
+
+In addition, it is generally faster than running equivalent simulations on the QVM, as the
+final state can be inspected and the expectation value calculated analytically, rather than by sampling measurements.
+
+
+.. note::
+
+    Since the NumPy wavefunction simulator is written entirely in NumPy, no external
+    Quil compiler is required.
+
+
+.. note::
+
+    By default, ``forest.numpy_wavefunction`` is initialized with ``shots=0``, indicating
+    that the exact analytic expectation value is to be returned.
```

Comment:
Wait, was the `analytic=True` == `shots=0` change made in PennyLane core? :thinking: Might have missed it

Reply:
I changed it to `analytic=True`, but this goes to show that we should check the device docs when releasing...

Reply:
@antalszava I think this block of text is so old, it is referring to the pre PL v0.3 way of doing things 😆